### ISSUE TITLE
Comments on "Quick and dirty fix for broken Button Icon Set Editor in MRTK 2.7.2"

### ISSUE THREAD

**Issue by LocalJoost** [LocalJoost/BlogComments#392]
Original article: https://localjoost.github.io/Quick-and-dirty-fix-for-broken-Button-Icon-Set-Editor-in-MRTK-272/

**Comment by icegibbon** [comment]
I ended up changing the inspector to Debug view. It worked fine for me.

**Comment by LocalJoost** [comment]
Yeah I've been told very recently (as in last week) that works as well. I prefer my solution - you yours. Whatever floats your boat, right ;)

**Comment by melixor** [comment]
You are a master. A solution for every issue.
